docs/source/opentelemetry.mdx

```diff
@@ -206,9 +213,39 @@ You can show the details of any operation and see a breakdown of its processing
 
 Our example telemetry configuration assumes that Zipkin is running locally, and that we want to process every span individually as it's emitted.
 
-To prepare for production, we probably at least want to conditionally set the `url` option for the `ZipkinExporter` and replace our `SimpleSpanProcessor` with a  `BatchSpanProcessor`.
+To prepare for production, we'll want to optimize performance by sending our traces to an [OpenTelemetry Collector](https://opentelemetry.io/docs/collector/) using the `OTLPTraceExporter` and replace our `SimpleSpanProcessor` with a `BatchSpanProcessor`.
+The `Collector` should be deployed as a local sidecar `Agent` to buffer traces before they're sent along to their final destination.
```

Comment:
What is `Agent`?  What is `Collector`? (I mean, specifically, they are code-fenced.) They're not explained here / used in the code?

If you're talking about them in a general sense, let's choose a different visual treatment.

Reply:
fixed